doc/v2-manifesto.md

```diff
@@ -0,0 +1,27 @@
+## What sucks about Squirrel v1's Design™
+
+* Squirrel.Windows and Squirrel.Mac are completely different
+  - This confuses the living fuck out of people
+
+* Squirrel.Mac is missing lots of features
+  - Delta updates
+  - Updates via static HTTPS (i.e. S3)
+  - Rollout updates 
+
+* Squirrel.Windows uses NuGet files
+  - Not worth it now that Electron is such a huge client
+  - NuGet library gives us some tooling but also we're pinned to their garbage
+  - NuGet compression / decompression library sucks out loud
+  - Squirrel.Mac just uses Basic-Ass Zip Files, much better
+
+* Squirrel.Mac relies on lots of Xcode libraries that bitrot
+  - RAC is cool, but Xcode breaks every library all the time
+  - Nobody can build Squirrel.Mac anymore
```

Comment:
You may have looked into this already, but it'd be interesting to try to "rebase" squirrel.mac on top of https://sparkle-project.org/ in v2. It's actively maintained, has a bit more of the expected Mac UI out-of-the-box and supports a couple of the things from the "missing" section. I guess it wouldn't move Squirrel.Mac and Squirrel.Windows any closer to parity though... 🙃